ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog for githash
 
+## 0.1.6.4
```

Comment:
Since this exposes a new identifier, this should be a minor version bump to 0.1.7.0. Mind making that change?

Reply:
Done